gax/src/test/java/com/google/api/gax/rpc/OperationCallableImplTest.java

```diff
@@ -1111,7 +1112,8 @@ private void assertFutureCancelMetaCancel(OperationFuture<Color, Currency> futur
     assertThat(future.getMetadata()).isSameInstanceAs(future.getMetadata());
     assertThat(exception).isNotNull();
     assertThat(future.getMetadata().isDone()).isTrue();
-    assertThat(future.getMetadata().isCancelled()).isTrue();
+    // The metadata future will always be resolved immediately once the initial request resolves.
+    assertThat(future.getMetadata().isCancelled()).isEqualTo(!initialDone);
```

Comment:
Can this be either

assertFalse(future.getMetadata().isCancelled())

or assertTrue(future.getMetadata().isCancelled())